### ISSUE TITLE
Loop after download archive.

### ISSUE THREAD

**Issue by NIM-Pro** [nwjs/nw-builder#206]
Hello. Today I updated it and found that after downloading the archive to cache, node process goes to loop.
OS: Win7 x64.
Options:
```javascript
{
  files: '.\\yardstudio\\**',
  platforms: [ 'win64' ],
  currentPlatform: 'win64',
  cacheDir: '.\\yards-cli\\cache'
}
```
Output:
```
Latest Version: v0.12.0
Using v0.12.0
Create cache folder in C:\Users\NotIntMan\Documents\GitHub\yards-cli\cache\0.12.
0
Downloading: http://dl.nwjs.io/v0.12.0/nwjs-v0.12.0-win-x64.zip
  downloading [====================] 100% 0.0s
```
That's all. That's all I got.

**Comment by NIM-Pro** [comment]
One more thing. I checked the cache folder.
```
└───0.12.0
    └───win64
        │   credits.html
        │   d3dcompiler_47.dll
        │   ffmpegsumo.dll
        │   icudtl.dat
        │   libEGL.dll
        │   libGLESv2.dll
        │   nw.exe
        │   nw.pak
        │   nwjc.exe
        │   pdf.dll
        │
        ├───locales
        └───nwjs-v0.12.0-win-x64
            │   credits.html
            │   d3dcompiler_47.dll
            │   ffmpegsumo.dll
            │   icudtl.dat
            │   libEGL.dll
            │   libGLESv2.dll
            │   nw.exe
            │   nw.pak
            │   nwjc.exe
            │   pdf.dll
            │
            └───locales
                    am.pak
                    etc...
```
Files in the folder nwjs-v0.12.0-win-x64 is not damaged, but they were not copied to the right place.

**Comment by adam-lynch** [comment]
I think this might be resolved by #337 and version 2.2.1. Thanks!
